index.html

```diff
@@ -31,15 +28,48 @@
         <link href="css/custom-blog.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/bootstrap-theme.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
-        <script>
-     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-     ga('create', 'UA-17146009-5', 'auto');
-     ga('send', 'pageview');
-    </script>
+	
+		<script type="text/javascript">
+
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-17146009-5', 'auto');
+    ga('send', 'pageview');
+    
+	//Captcha	
+	function Captcha(){
+			var alpha = new Array('1','2','3','4','5','6','7','8','9','0','A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z','a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z');
+			var i;
+			for (i=0;i<6;i++){
+			  var a = alpha[Math.floor(Math.random() * alpha.length)];
+			  var b = alpha[Math.floor(Math.random() * alpha.length)];
+			  var c = alpha[Math.floor(Math.random() * alpha.length)];
+			  var d = alpha[Math.floor(Math.random() * alpha.length)];
+			  var e = alpha[Math.floor(Math.random() * alpha.length)];
```

Comment:
Just add reCaptcha and not regular captcha.

Reply:
Ok, will work on it.